src/components/InjectedComponents/ImmersiveSetup/SetupStepper.tsx

```diff
@@ -172,43 +178,36 @@ export function ImmersiveSetupStepperUI(props: ImmersiveSetupStepperUIProps) {
                         />
                         <br />
                         <Typography>{t('immersive_setup_username_confirm')}</Typography>
-                        <Typography>
-                            <Link
-                                target="_blank"
-                                rel="noopener noreferrer"
-                                color="textSecondary"
-                                href="https://maskbook.com/faq/?2"
```

Comment:
does this link broken?

Reply:
I don't think so

Reply:
> does this link broken?

it's working

Reply:
The linked content is not explaining `immersive_setup_help_dont_know_what_is_username(But I don't know what my username is!)`.

Reply:
Reference internal conversation: https://dimension.chat/group/maskbook?msg=2NZzYMhRX4tbB7TPX

Reply:
@septs proposed to standardize our link shortener with git.io. What do you guys think? especially @neruthes

Reply:
we can leave this to another issue and let this pr pass

Reply:
> we can leave this to another issue and let this pr pass

ok

Reply:
<img width="564" alt="image" src="https://user-images.githubusercontent.com/2139261/74997528-35803b80-544e-11ea-94d3-dcaffb833189.png">

I feel difficult to get the point of why \<git.io> may be a favorable solution.

And I would like to explain that the requirement for short linking is not to reduce the length of the URL, but to update the value of `href` without updating the software. In other words, the purpose is to move the get_final_url operation from client to server, since FAQ is more like a web service instead of a software feature.

cc @yisiliu